### PR TITLE
FIXED URLs that were redirecting in the README

### PR DESCRIPTION
**Pull request by frankenbot**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

**Comment by kcharwood** [comment]
Thanks!

**Comment by dkhamsing** [comment]
:smile:

File: README.md [hunk 1]
```diff
@@ -360,7 +360,7 @@ NSArray *operations = [AFURLConnectionOperation batchOfRequestOperations:@[...]
 
 ## Unit Tests
 
-AFNetworking includes a suite of unit tests within the Tests subdirectory. In order to run the unit tests, you must install the testing dependencies via [CocoaPods](http://cocoapods.org/):
+AFNetworking includes a suite of unit tests within the Tests subdirectory. In order to run the unit tests, you must install the testing dependencies via [CocoaPods](https://cocoapods.org//):
```
Comment (kylef) [inline README.md]: This introduced a double slash in the end of the URI. /cc @dkhamsing

**Comment by dkhamsing** [comment]
Oh no

**Pull request merged**